src/AdoPat/AdoPat.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Authentication.AdoPat</PackageId>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>

```

Comment:
Why are we not targeting net6.0?

Reply:
That's a great question. I had actually just copied from other files we have in this repo here. @kyle-rader: Any reason we shouldn't just target `net6.0` here?

Reply:
This is a fantastic question! Dotnet run-times are implementations of Dotnet standards (until you reach dotnet 5). A Library wanting to maximize compatibility, should always target a dotnet standard version (2.0 or 2.1), while an executable must choose a runtime version. [Dotnet Standard](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-1#net-5-and-net-standard) has a ton of great info here. From reading more on this just now since I hadn't read that page in a little while,

Dotnet standard 2.1 is supported by Dotnet 5+ and will continue to be. This means we can always use libraries still targeting that standard. But, As of dotnet 5 - the dotnet platform is consolidated (no core vs framework) and so within our own app - we can target net 5/6 directly in our libraries, and while that makes them only usable by net6 applications, this is fine, because it's only auzreauth using them. This opens up the door to use using C# 9!

So @reillysiemens I do think we should go ahead and target net6 directly here.

Reply:
Done. This now targets net6.